### PR TITLE
Fix tally

### PR DESCRIPTION
**Pull request by TApplencourt**
Sorry for all the other typo, it's clang-format

Focus on `_get_uniq_tally`

Thanks @nliber

### CODE REVIEW

**Comment by Kerilk** [comment]
Is @colleeneb example summary fixed using this patch?

**Comment by TApplencourt** [comment]
Yep! 
```
Explicit memory traffic (BACKEND_OMP) | 16 Hostnames | 192 Processes | 192 Threads |
Explicit memory traffic (BACKEND_ZE) | 16 Hostnames | 384 Processes | 384 Threads |
```
Which is expected for this app (and 16*24 = 384 so we are good)

File: xprof/btx_tally.hpp [hunk 1]
```diff
@@ -139,28 +139,40 @@ auto aggregate_by_name(std::unordered_map<std::tuple<T...>, TC> &m) {
...
-template<typename... K, typename V>
-auto get_uniq_tally(std::unordered_map<std::tuple<K...>, V> &input) {
-  auto tuple_set = std::make_tuple(std::set<K>{}...);
-  constexpr auto s = std::make_index_sequence<sizeof...(K)>();
 
-  for (auto const &m : input)
-    add_to_set(tuple_set, m.first, s);
-  return tuple_set;
+// Base case to stop recursion
+inline void _get_uniq_tally(std::set<std::tuple<>>, std::vector<size_t> &) {}
```
[the remark this answers is not in the document]
Comment (TApplencourt) [reply xprof/btx_tally.hpp]: Totaly agree! Fixed with 9ee7fac

**Comment by colleeneb** [comment]
> Is @colleeneb example summary fixed using this patch?

Yep, as @TApplencourt showed, it looks correct!

**Review by nliber — APPROVED** [review]

**Review by Kerilk — APPROVED** [review]

**Pull request merged**